etna/lib/etna/parse_body.rb

```diff
@@ -23,7 +23,7 @@ def call(env)
         )
       when %r{multipart/form-data}i
         params.update(
-          Rack::Multipart.parse_multipart(env)
+          Rack::Multipart.parse_multipart(env) || {}
```

Comment:
When there is no multipart data (but the `multipart/form-data` header is included), this part would throw an exception trying to update the parameters because `Rack::Multipart.parse_multipart()` would return `nil`. Really that should only happen in a test environment, but this addition lets the process continue cleanly.

Reply:
Hmm why is this only possible in a test environment?

Reply:
I guess I would only expect it to happen in a test environment (or if someone intentionally constructs such a message). I wouldn't expect something like that to happen within one of our applications though.